### ISSUE TITLE
browser_mod.set_theme doesn't work anymore

### ISSUE THREAD

**Issue by gamer06** [thomasloven/hass-browser_mod#141]
Hello.

I'am with 0.155.6 version and when i try execute this commands below and now doesn't work anymore.

"
script:
  theme_clear1_nexus12:
    sequence:
      - service: browser_mod.set_theme
        data:
          theme: clear1
          deviceID:
            - nexus12
"

or:

https://gyazo.com/069b69ad4644f03c2a75d8b23787b851

Thanks

**Comment by KTibow** [comment]
Please thoroughly check the previous issues.

**Comment by gamer06** [comment]
> Please thoroughly check the previous issues.

I read the other issues and found no solution ...

**Comment by KTibow** [comment]
Check the forum thread.

**Comment by gamer06** [comment]
I didn't find the solution. Could you point me out?

**Comment by KTibow** [comment]
Someone will do a PR sooner or later. @thomasloven doesn't even watch this repo for notifications anymore.

**Comment by gadgetchnnel** [comment]
It looks like this may have broken when light/dark mode support was added - Home Assistant expects the selected theme to be an object (like {"theme":"theme name"}) rather than just a string. Changing the service call like this seems to work:

```
script:
  theme_clear1_nexus12:
  sequence:
    - service: browser_mod.set_theme
      data:
        theme:
          theme: clear1
        deviceID:
          - nexus12
```

**Comment by KTibow** [comment]
Docs PR: #149

**Comment by thomasloven** [comment]
Fixed by #149
